<launch>
	<!-- start the harriet model -->
	<include file="$(find model_harriet)/launch/model_harriet_no_display.launch"/>

	<!-- start the ball detector with some fun parameters -->
	<node pkg="tennis_ball_detection" type="ball_detector" name="ball_detector_node" output="screen">
	      <param name="display" type="string" value="local"/>
	      <param name="ball_output_frame" type="string" value="/main_body"/>
	      <param name="do_continuous" type="bool" value="true"/>
	</node>
</launch>